rmw_dds_common/msg/NodeEntitiesInfo.idl

```diff
@@ -0,0 +1,14 @@
+#include "rmw_dds_common/msg/Gid.idl"
+
+module rmw_dds_common {
+  module msg {
+    struct NodeEntitiesInfo {
+        @key
+        string<256> node_namespace;
+        @key
```

Comment:
Well, these two keys actually don't contribute in anything.

---

Originally, I planned to use keyed topics (which would be a bit more efficient).
Then I realized that I would actually need to add support for that in `rosidl_typesupport_fastrtps`, which is completely out of scope.
So, I'm not using a keyed topic at all. But I think that the `@key` in `ParticipantEntitiesInfo.idl` is fine (just in case in the future we have support for them).

Reply:
deleted those keys